travispy/entities/log.py

```diff
@@ -1,23 +1,38 @@
 from ._entity import Entity
 
+from travispy._helpers import get_archived_log
+
 
 class Log(Entity):
     '''
     :ivar int job_id:
         Jod ID.
 
-    :ivar str body:
-        Log body.
-
     :ivar str type:
     '''
 
     __slots__ = [
         'job_id',
-        'body',
+        '_body',
         'type',
     ]
 
+    def __init__(self, session):
```

Comment:
Oh, I see. Even ``body`` is archived (which means an empty body) requesting it as plain text will return the last log text. Is that it? If that is the case keep these changes.

Related to ``get_archived_log`` here, I still think that is the best option.